modules/installation-gcp-shared-vpc-config.adoc

```diff
@@ -3,68 +3,23 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="installation-gcp-shared-vpc-config_{context}"]
-= Sample customized install-config.yaml file for shared VPC installation
-There are several configuration parameters which are required to install {product-title} on GCP using a shared VPC. The following is a sample `install-config.yaml` file which demonstrates these fields.
-
-[IMPORTANT]
-====
-This sample YAML file is provided for reference only. You must modify this file with the correct values for your environment and cluster.
-====
+= Enabling installation into a shared VPC
+There are several configuration parameters which are required to install {product-title} on GCP using a shared VPC.
+The following parameters must be added to the `install-config.yaml` file prior to installation.
 
+.Shared VPC parameters in `install-config.yaml`
 [source,yaml]
 ----
-apiVersion: v1
-baseDomain: example.com
-credentialsMode: Passthrough <1>
-metadata:
-  name: cluster_name
 platform:
   gcp:
-    computeSubnet: shared-vpc-subnet-1 <2>
-    controlPlaneSubnet: shared-vpc-subnet-2 <3>
-    network: shared-vpc <4>
-    networkProjectID: host-project-name <5>
-    projectID: service-project-name <6>
-    region: us-east1
-    defaultMachinePlatform:
-      tags: <7>
-      - global-tag1
-controlPlane:
-  name: master
-  platform:
-    gcp:
-      tags: <7>
-      - control-plane-tag1
-      type: n2-standard-4
-      zones:
-      - us-central1-a
-      - us-central1-c
-  replicas: 3
-compute:
-- name: worker
-  platform:
-    gcp:
-      tags: <7>
-      - compute-tag1
-      type: n2-standard-4
-      zones:
-      - us-central1-a
-      - us-central1-c
-  replicas: 3
-networking:
-  clusterNetwork:
-  - cidr: 10.128.0.0/14
-    hostPrefix: 23
-  machineNetwork:
-  - cidr: 10.0.0.0/16
-pullSecret: '{"auths": ...}'
-sshKey: ssh-ed25519 AAAA... <8>
+    computeSubnet: shared-vpc-subnet-1 <1>
+    controlPlaneSubnet: shared-vpc-subnet-2 <2>
+    network: shared-vpc <3>
+    networkProjectID: host-project-name <4>
+    projectID: service-project-name <5>
 ----
-<1> `credentialsMode` must be set to `Passthrough` or `Manual`. See the "Prerequisites" section for the required GCP permissions that your service account must have.
-<2> The name of the subnet in the shared VPC for compute machines to use.
-<3> The name of the subnet in the shared VPC for control plane machines to use.
-<4> The name of the shared VPC.
-<5> The name of the host project where the shared VPC exists.
-<6> The name of the GCP project where you want to install the cluster.
-<7> Optional. One or more network tags to apply to compute machines, control plane machines, or all machines.
-<8> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
\ No newline at end of file
+<1> The name of the subnet in the shared VPC for compute machines to use.
```

Comment:
The `credentialsMode` still applies I believe. Should be leave that or mention it somewhere else ?

Reply:
I was going off of what we have the parameters table, which is "If no mode is specified, the CCO dynamically tries to determine the capabilities of the provided credentials, with a preference for mint mode on the platforms where multiple modes are supported. " 

would you recommend we still specify credentials mode? if so, which mode should we put in the sample?